wordpress-objects.taxonomy.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+class Taxonomy {
+
+	protected $_tax;
+
+	public static function get( $taxonomy ) {
```

Comment:
@mattheu the singleton getter is meant to initiate the object and return it - this is going to throw a fatal as you are using `$this` in the static context.

```PHP
public static function get( $taxonomy ) {
    // do error checking first
    return new Taxonomy( $taxonomy );
}

public function __construct( $tax ) {
    $this->_tax = $tax;
}
```

There is a couple of reasons for this: 1: the static `get` can return a WP_Error (calling __construct directly can't return a wp error). Also, the `Taxonomy::get( 'post-tag' )` indicates you are getting an existing taxonomy, and not "creating" a new one.